src/strongbox/nfo.clj

```diff
@@ -54,7 +63,9 @@
 
              ;; where the addon came from and how they identified it
              :source (:source addon)
-             :source-id (:source-id addon)}
+             :source-id (:source-id addon)
+
+             :source-map-list [{:source (:source addon) :source-id (:source-id addon)}]}
```

Comment:
what happens when we `derive` from derived data? we'd lose whatever `source-map-list` already was ... do we do that anywhere? Is it unreasonable to do so?

Reply:
add note about how nfo only captures sources that have been *installed*, not the list of all known ones that comes from the toc data

Reply:
the only place `derive` is used is during addon installation. Thereafter `update-nfo` is used. We're safe for now.